pkg/util/common.go

```diff
@@ -60,3 +63,11 @@ func Keys[K comparable, V any](m map[K]V) []K {
 	}
 	return keys
 }
+
+func PrintTimeCost(fuc string) func() {
+	t := time.Now()
+	return func() {
+		cost := time.Since(t)
+		klog.Infof("[PrintTimeCost] %s cost %+v", fuc, cost)
```

Comment:
We need to adjust the log printing level, either 4 or 5 would be more appropriate.

Reply:
You can see I have tried 5 level in previous force push, but it doesn't work, I didn't see the log output.